libs/sdk-backend-tiger/src/convertors/fromBackend/afm/dimensions.ts

```diff
@@ -54,6 +57,10 @@ function transformDimension(dim: Execution.IResultDimension): IDimensionDescript
                                     localIdentifier: m.measureHeaderItem.localIdentifier,
                                     name: m.measureHeaderItem.name ?? m.measureHeaderItem.localIdentifier,
                                     format: m.measureHeaderItem.format ?? DEFAULT_FORMAT,
+                                    identifier: m.measureHeaderItem.identifier,
+                                    ref: m.measureHeaderItem.identifier
+                                        ? idRef(m.measureHeaderItem.identifier, "measure")
```

Comment:
Can this ever be a `"fact"`?

Reply:
good catch. looking into tiger backend code, i see it does not even return identifier. to match the SPI contract, i'm going to augment the descriptor with identifier & ref used on the execution definition.